examples/pytorch/dimenet/modules/bessel_basis_layer.py

```diff
@@ -17,12 +17,14 @@ def __init__(self,
         self.reset_params()
 
     def reset_params(self):
-        torch.arange(1, self.frequencies.numel() + 1, out=self.frequencies).mul_(np.pi)
+        with torch.no_grad():
+            torch.arange(1, self.frequencies.numel() + 1, out=self.frequencies).mul_(np.pi)
+        self.frequencies.requires_grad_()
```

Comment:
Is L22 necessary here? Have you checked `self.frequencies.requires_grad` at L22?